### ISSUE TITLE
make rust upgrade using autobot

### ISSUE THREAD

**Issue by gengjiawen** [gitpod-io/workspace-images#1326]
## Is your feature request related to a problem? Please describe

currently this is still manual, in the long run this should be done via auto bot

## Describe the behaviour you'd like

use current bot implement this.

**Comment by filiptronicek** [comment]
I agree, we should add another fixer in https://github.com/gitpod-io/workspace-images/tree/f152b0380284358874d93913ef6217e7fb1b437a/.autofix/fixers, based off of GitHub Releases / https://endoflife.date.

**Comment by filiptronicek** [comment]
Fixed in https://github.com/gitpod-io/workspace-images/pull/1441
